app/src/main/java/cm/aptoide/pt/view/MainActivity.java

```diff
@@ -205,6 +224,30 @@ private void setupUpdatesNotification() {
     }
   }
 
+  public void initializeMoPub() {
+    SdkConfiguration sdkConfiguration = new SdkConfiguration.Builder(
+        BuildConfig.MOPUB_BANNER_50_HOME_PLACEMENT_ID).withAdditionalNetwork(
+        AppLovinBaseAdapterConfiguration.class.toString())
+        .withMediatedNetworkConfiguration(AppLovinBaseAdapterConfiguration.class.toString(),
+            getMediationNetworkConfiguration(BuildConfig.MOPUB_BANNER_50_HOME_PLACEMENT_ID))
+        .withAdditionalNetwork(AppnextBaseAdapterConfiguration.class.toString())
+        .withMediatedNetworkConfiguration(AppnextBaseAdapterConfiguration.class.toString(),
+            getMediationNetworkConfiguration(BuildConfig.MOPUB_BANNER_50_HOME_PLACEMENT_ID))
+        .withMediatedNetworkConfiguration(InMobiBaseAdapterConfiguration.class.toString(),
+            getMediationNetworkConfiguration(BuildConfig.MOPUB_BANNER_50_HOME_PLACEMENT_ID))
+        .withLogLevel(MoPubLog.LogLevel.DEBUG)
+        .build();
+
+    MoPub.initializeSdk(this, sdkConfiguration, null);
+  }
+
+  @NonNull
+  private Map<String, String> getMediationNetworkConfiguration(String mediatedNetworkPlacementId) {
+    Map<String, String> mediationNetworkConfiguration = new HashMap<>();
+    mediationNetworkConfiguration.put("Placement_Id", mediatedNetworkPlacementId);
+    return mediationNetworkConfiguration;
+  }
+
```

Comment:
Would keep this initialization in application since it was working and it is risky to delay this.